load_blazegraph_test.sh

```diff
@@ -10,16 +10,85 @@ BASEURI="http://ld4p-test.stanford.edu/"
 # vars above this line may need to change to process other data
 #------------------------------------------------
 
-BG_URL="http://localhost/blazegraph/"
-TEST_NAMESPACE=test
-BG_SPARQL="${BG_URL}/namespace/${TEST_NAMESPACE}/sparql";
 
-RDF_FILE="test_data/86169543_rdf.xml"
+# ---
+# Detect Blazegraph settings or use defaults
 
-log_stamp=$(date --iso-8601=sec)
-echo "${log_stamp}  Uploading ${RDF_FILE} into ${BG_SPARQL}"
+if [ -f /etc/default/blazegraph ]; then
+    # Blazegraph is installed as a system service
+    # Source this file, which contains a JETTY_PORT setting
+    source /etc/default/blazegraph
+fi
 
-curl -s -X POST -H 'Content-Type:application/rdf+xml' --data-binary "@${RDF_FILE}" ${BG_SPARQL}
+if [ -z "$JETTY_PORT" ]; then
+    if [ -d /usr/share/tomcat/webapps/blazegraph ]; then
+        # Blazegraph is installed as a tomcat servlet
+        # Assume this is running on the main HTTP port
+        JETTY_PORT=80
+    fi
+fi
+
+if [ -z "$JETTY_PORT" ]; then
+    # Try the Blazegraph default port
+    JETTY_PORT=9999
+fi
+
```

Comment:
This `JETTY_PORT` stuff is confusing and appears to only be a concern for your laptop dev environment. Can you just override a `PORT` var when you're running it locally? Furthermore, we should probably default the port to straight to tomcat on 8080 instead of using apache's proxy.